config/default.conf

```diff
@@ -54,7 +54,7 @@ server {
 
 	location	/cgi-bin/ {
 		client_max_body_size 100000000;
-		limit_except POST GET;
+		limit_except POST GET; #GET GET - duplicates work. Should they?
```

Comment:
Duplicates work in nginx too. I think it is up to us to decide if we want to allow them or not. I don't have anything against them.

Reply:
I've added some more unit tests